src/ResourceManager/Compute/Commands.Compute/Az.Compute.psd1

```diff
@@ -147,8 +147,7 @@ CmdletsToExport = 'Remove-AzAvailabilitySet', 'Get-AzAvailabilitySet',
     'New-AzDiskConfig', 'Set-AzDiskDiskEncryptionKey', 
     'Set-AzDiskImageReference', 'Set-AzDiskKeyEncryptionKey', 
     'New-AzDiskUpdateConfig', 
-    'Set-AzDiskUpdateDiskEncryptionKey', 
-    'Set-AzDiskUpdateImageReference', 
```

Comment:
what happened to these imagereference cmdlets?

Reply:
They appear to have been taken out a while ago in AzureRM but were never removed in Az.  I caught this when trying to generate the index.json file, as the help files/cmdlet code was removed when the cmdlets were taken out of AzureRM.